tests/e2e/collectives/CMakeLists.txt

```diff
@@ -11,49 +11,108 @@ if(NOT IREE_BUILD_BUNDLED_LLVM OR NOT IREE_ENABLE_COLLECTIVE_RUNTIME_TESTS)
 endif()
 
 if(IREE_TARGET_BACKEND_CUDA AND IREE_HAL_DRIVER_CUDA)
+
+  set(COMMON_ARGS
+    "--target_backend=cuda"
+    "--driver=cuda"
+    "--iree_compiler_args=--iree-hal-cuda-llvm-target-arch=sm_53"
+  )
+
+  set(COMMON_LABELS
+    "requires-gpu-amd"
+    "driver=hip"
+  )
```

Comment:
Why does the `if(IREE_TARGET_BACKEND_CUDA AND IREE_HAL_DRIVER_CUDA)` block require an AMD GPU?

Some comments around each block of tests would help. "CUDA on NVIDIA", "HIP (CUDA) on AMD", "HIP (ROCm) on AMD", etc.? I'm confused by the different APIs and configuration choices here :P

Reply:
Good catch. I made a find-replace error.  I fixed it.